NewYorkTimes-Article-Page

elasfarc opened the pull request:
In this project we worked on the following:
	•	Created HTML body & CSS styles.
	•.     Create a fixed Navbar
	•.     Added layouts in Grid, Flex, position and Float
	•.     Added hovers to actionable links.
	•.     Added relevant images, videos, links and icons.

elasfarc committed:
init

init index.html
style.css
.github/workflow/linters

elasfarc committed:
Create linters.yml

elasfarc committed:
nav,aside,article till YT video

<html>
structure the nav , aside,start with the article till the youtube video

elasfarc committed:
style main nav/AD section

html -> adjust the main structure instead of wraping everything sepratly

css-> style the main nav
           style the AD section class

elasfarc committed:
style article-main-image-caption

style main header/main image & caption

Fanger53 committed:
fix fig and author box

elasfarc committed:
fix main structure , style article

html:
-default width of wrapper -> 100%
-remove unnecessary div around main title h1 and apply the class to it's parent dic
-seperate the author class-social-btn div from the main-title-fig

style class article

Fanger53 committed:
adding border and making corrections to the aside

Fanger53 committed:
aside configuration

elasfarc committed:
style the inflationBOX class

css:
style the inflationBOX class
html:structure the rest of the file till the suggested articles

Fanger53 committed:
aside editor pic put image

Fanger53 committed:
Merge branch 'article-clone' of https://github.com/elasfarc/NewYorkTimes-article-clone into article-clone

Fanger53 committed:
finishing editor's pics and fix his position

Fanger53 committed:
adding footer, footer sections

elasfarc committed:
complete the html structure till the footer

complete the html structure till the footer

Fanger53 committed:
merge fix clonflicts

elasfarc committed:
suggested articles DIV

html:
-fix previous merge conflicts
-structure the suggested articles DIV

css:
set the grid for suggested-articles
style suggested-articles-aside
flex suggested-articles-section

Fanger53 committed:
adding elements into the footer

Fanger53 committed:
adding footer elements

elasfarc committed:
style article

style #subscribe div inside second article
increase width for the whole article class
add <a> tags / style

elasfarc committed:
main aside

style main-aside class

elasfarc committed:
add class article rights

structure&style
YT-video , article-rights

Fanger53 committed:
fixing style commet-share box, margins for the footer

Fanger53 committed:
Update README.md

elasfarc committed:
fix styling for the suggested articles

seprate font.css
style & position main-nav
fix main figure

elasfarc committed:
Merge branch 'article-clone' of https://github.com/elasfarc/NewYorkTimes-article-clone into article-clone

Fanger53 committed:
fix horizontal scroll-bar from the footer

elasfarc committed:
style main footer

fix main-footer postioning / main-aside clickable / YT video width

Fanger53 committed:
adding linters and fix css style

elasfarc committed:
stylelint

add generic font family

Fanger53 committed:
styles linters, just expected issues

elasfarc committed:
fix descending specificity

moving styling for author id to be before the subscrible id styling

Fanger53 committed:
left 8 linters errors

elasfarc committed:
fixing linters error

fixing linters error
descending-specificity

Fanger53 committed:
style css file all linters are off

elasfarc committed:
minor

fixing main-footer icons relative path
main-footer header border top margin
fixing fonts.css linters errors
add DS_store to .gitignore

elasfarc committed:
HTML checker

remove consecutive hyphens in HTML comments

elasfarc committed:
styling the README.md

styling the README.md
fixing postioning

elasfarc committed:
change units measurements

change units measurements from px to vw

elasfarc committed:
Edit README.md

Add a screenshot

elasfarc committed:
README.md

edit README.md

elasfarc committed:
Update GHP-SCSH.png

elasfarc committed:
Edit date-share icons size

Edit date-share icons size

elasfarc committed:
Update GHP-SCSH.png

elasfarc committed:
Edit mark Icon size

Edit date-share Icons size in class date-share

elasfarc committed:
Update GHP-SCSH.png

elasfarc committed:
fixing Stylelint errores

removing whiite spaces
remove font-display property from fonts.css

elasfarc committed:
fixing stylelint errores

elasfarc committed:
make main-logo a link

make the main-logo a link to the homepage https://www.nytimes.com/

elasfarc committed:
README.md

Edit demo-link in README

elasfarc committed:
Edit demo-link in README

elasfarc committed:
Update index.html

peterrobert commented on index.html:
```diff
@@ -0,0 +1,876 @@
+                        </ul>
+
+
+                     <!--  <i class="far fa-comment-alt " id="icon-box" ><span id="icon-num" ></span></i>-->
```
kindly remove commented code.

peterrobert commented on index.html:
```diff
@@ -0,0 +1,876 @@
+                </span>
+            </div>
+            <div class="inflationDIV" >
+             <!--<img id="img-in" src="./assets/inflation-720.png">-->
+               <div class="row1p">
```
kindly remove commented code it keeps your work clean

peterrobert commented on assets/stylesheet/styles.css:
```diff
@@ -0,0 +1,824 @@
+
+/* YT figure */
+
+#YT-video {
+  margin-block-start: 0;
+  margin-block-end: 0;
+  margin-inline-start: 0;
+  margin-inline-end: 0;
+}
```
There are a few CSS errors at this point kindly fix them.
![warnings css](https://user-images.githubusercontent.com/32361963/88902034-4d0feb80-d25a-11ea-9a8e-e4ffc9257876.jpg)

peterrobert reviewed (changes requested):
### CHANGES REQUIRED :recycle: :stop_sign: 
### HELLO @elasfarc AND @Fanger53  :wave:  AM PETER ROBERT A TSE AND I HAVE REVIED YOUR PROJECT.
### YOUR PROJECT LOOKS AMAZING FOR A FIRST REVIEW. IT ONLY HAVE SOME FEW CHANGES TO BE MADE. KINDLY CHCKE THE CHANGES BELOW :arrow_down: 

- [x] Kindly add a space logo to your project. Just like in the original :arrow_down: 
### original
![space logo](https://user-images.githubusercontent.com/32361963/88902743-48980280-d25b-11ea-8fd5-4246ce31d26c.jpg)

- [x] Kindly you need to add a fixed footer to your project just like in the original :arrow_down: 
### original
![fixed footer](https://user-images.githubusercontent.com/32361963/88902893-75e4b080-d25b-11ea-9771-90751bc7f5e2.jpg)

### [OPTIONAL]
Your HTML has a lot of warnings. Since they are not errors it is not a requirement to fix them.
![warnings](https://user-images.githubusercontent.com/32361963/88903031-b0e6e400-d25b-11ea-83f8-5b10edf3ea1c.jpg)

When you are done kindly submit your project for another review.

### if you have a question concerning the review kindly reach me via slack on @peterrobert

elasfarc committed:
fixed-footer

add fixed-footer and .section into .main-nav

Fanger53 committed:
last changes

elasfarc committed:
fix the fixed-footer so its fixed

Fanger53 committed:
fix errors

Fanger53 committed:
fixing linters errors from hint and style

Fanger53 committed:
Update README.md

expjazz reviewed (approved):
## Status: Project Approved :green_circle: 

Your clone looks awesome :clap: 

However, you still have a commented code on line 243. Since you checked all the minimum requirements and removed other ones that the previous reviewer pointed out, I'm not going to reprove it because I'm assuming that you missed this one. Kindly remove it because it is not a good practice.

You did great on this :100: 

Good Luck and Happy Coding :pray:

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Expedito Andrade).:smile:_

elasfarc committed:
fixed-footer

-adding a fixed-footer-wraper to fixed-footer with min-height
-removing commented code.
